build-pelux.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
I think we should have the instructions on how to build and use PELUX Baseline in the SWF. I think we should avoid having multiple locations due to the increased maintenance burden. https://pelagicore.github.io/software-factory/chapters/baseplatform/index.html